src/styles/blocks/header.scss

```diff
@@ -0,0 +1,60 @@
+.header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+
+  .logo {
+    height: 40px;
+    width: 40px;
+    object-fit: contain;
+    padding: 10px 0;
+  }
+
+  nav {
+    text-transform: uppercase;
+
+    .nav-list {
+      display: flex;
+      margin: 0;
+      padding: 0;
+      list-style: none;
+
+      .nav-item {
+        transition: all ease-in-out 0.15s;
+        color: $nav-link-color;
+
+        &:not(:last-child) {
```

Comment:
The `margin-right` value for `.nav-item` should be a multiple of 4. Consider adjusting it to adhere to the code style guide.